### PR TITLE
Professor now given a propmt when the change the visibility of a quesiton.

### PR DESCRIPTION
**Pull request by Mohamed-A-Mohamed**
This PR fixes issue #99 
anim

### CODE REVIEW

File: public/js/admin.js [hunk 1]
```diff
@@ -474,34 +474,60 @@ var submitEditForm = function(id) {
...
+        closeOnConfirm: false,
+        closeOnCancel: false,
         },
-        error: function(data){
-            if (data['status'] === 401) {
-                window.location.href = '/';
+        function (isConfirm) {
+            // User confirms the visiblity change
+            if (isConfirm) {
+            swal('Changed!', 'The visibility of the quesiton has been changed', 'success');
```
Comment (esteeroramy) [inline public/js/admin.js]: I don't think we should have that secondary SWAL. A Snackbar notif should be enough. (Less clicks are more convenient for the user.

**Review by esteeroramy — CHANGES_REQUESTED** [review]

File: public/js/admin.js (hunk 1 above)
Comment (Mohamed-A-Mohamed) [reply public/js/admin.js]: I agree when the user confirms the change, but what if they cancel. There is no snack bar notification, so I think I should keep the swal after the user cancels. What do you think?

Comment (ahmedqarmout2) [reply public/js/admin.js]: No need for anythjng when canceling.

File: public/js/admin.js [hunk 2]
```diff
@@ -474,34 +474,60 @@ var submitEditForm = function(id) {
...
+                    } else {
+                        displayQuestionTable();
+                        // Toast notification
+                        dropSnack(colours.FAIL, 'Could not change visibility of question');
+                    }
+                }
+            });
+            // User cancels the visibility change
             } else {
+                swal('Cancelled', 'The visibility of the question was not changed.', 'error');
```
Comment (ahmedqarmout2) [inline public/js/admin.js]: No need for this.

**Review by ahmedqarmout2 — CHANGES_REQUESTED** [review]

File: public/js/admin.js (hunk 1 above)
Comment (Mohammed-JS-Khan) [reply public/js/admin.js]: If you guys want you can make a swal that goes away after 2 seconds without the need for user input.

**Review by Mohammed-JS-Khan — APPROVED** [review]
Just need to figure out the whole double swal business.

File: public/js/admin.js (hunk 1 above)
Comment (mehtadarshan1) [reply public/js/admin.js]: The snack bar already notifies the admin that it was successful or not

**Review by mehtadarshan1 — APPROVED** [review]

**Review by esteeroramy — APPROVED** [review]

**Review by ahmedqarmout2 — APPROVED** [review]

**Pull request merged**